frontend/components/ToastNotification/_styles.scss

```
@@ -0,0 +1,251 @@
+      ),
+      $core-fleet-white;
+    color: $core-fleet-black;
+    border: 0;
+    border-radius: $border-radius-large;
+    box-shadow: $toast-notification-shadow;
+    box-sizing: border-box;
+    font-size: $x-small;
+    font-family: "Inter", sans-serif;
+    line-height: $line-height;
+
+    gap: 0;
```

Stylelint (17.12.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)